applications/ShapeOptimizationApplication/custom_utilities/damping/damping_utilities.cpp

```diff
@@ -0,0 +1,172 @@
+// ==============================================================================
+//  KratosShapeOptimizationApplication
+//
+//  License:         BSD License
+//                   license: ShapeOptimizationApplication/license.txt
+//
+//  Main authors:    Baumgaertner Daniel, https://github.com/dbaumgaertner,
+//                   Willer Matthias, https://github.com/matthiaswiller
+//
+// ==============================================================================
+
+// ------------------------------------------------------------------------------
+// System includes
+// ------------------------------------------------------------------------------
+#include <iostream>
+#include <string>
+#include <algorithm>
+
+// ------------------------------------------------------------------------------
+// Project includes
+// ------------------------------------------------------------------------------
+#include "custom_utilities/damping/damping_utilities.h"
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "spatial_containers/spatial_containers.h"
+#include "utilities/builtin_timer.h"
+#include "utilities/parallel_utilities.h"
+#include "custom_utilities/filter_function.h"
+#include "shape_optimization_application.h"
```

Comment:
this is only necessary as the Variables are declared in this file
Another cleanup would be to move the vars to a separate file as is done for the other apps :)

Reply:
you mean importing the application? Yes, variables can be moved to another file in another PR =)